manifests/cluster-install/workflow-controller-rbac/workflow-controller-clusterrole.yaml

```diff
@@ -71,4 +71,5 @@ rules:
   - watch
   - update
   - patch
-  - delete
\ No newline at end of file
+  - delete
+  - create
```

Comment:
Why does the workflow controller need delete or create ability on CronWorkflows?

Reply:
@simster7 I removed this permission, and 7 tests started to fail:

https://circleci.com/gh/argoproj/argo/1440

* TestCronSuite/TestBasic